### PR TITLE
Merge 6.x -> main

### PR DESCRIPTION
**Pull request by Magisus**

### CODE REVIEW

**Comment by justinstoller** [comment]
Shall we merge https://github.com/puppetlabs/puppetserver/pull/2401 up to 6.x before doing this?

**Comment by Magisus** [comment]
Sure, yeah, that's probably going to have conflicts. I'll get that done.

**Comment by Magisus** [comment]
Updated with second mergeup.

File: locales/puppetserver.pot [hunk 1]
```diff
@@ -252,11 +252,15 @@ msgid "Removed certificate request for {0} at ''{1}''"
 msgstr ""
 
 #: src/clj/puppetlabs/puppetserver/certificate_authority.clj
-msgid "Revoked {0} certificate with serial {1}"
+msgid "Revoked {0} certificate."
```
Comment (justinstoller) [inline locales/puppetserver.pot:255]: We decided that this was taken care of other automation, right?

Comment (Magisus) [reply locales/puppetserver.pot:255]: Yeah the POT file should get overwritten as part of the Jenkins pipeline.

**Comment by mwaggett** [comment]
Travis is lying, it looks green when you click in

**Pull request merged**